SixTrack/sixtrack.s

```diff
@@ -18428,7 +18428,7 @@ cc2008
       write(4,*) ch
       goto 3
  90   continue
-10000 format(a16,1x,i2,1x,d21.15,1x,d21.15,1x,d16.10)
+10000 format(a16,1x,i2,1x,d22.15,1x,d22.15,1x,d17.10)
```

Comment:
This is anyway just making the fields slightly wider, fields which are guaranteed to be separated by a space (the `1x` thing in the FORMAT). Unless someone is splitting on character counts, there should not be any issue.